main.py

```diff
@@ -656,6 +656,9 @@ def ui_access_wrapper(self, *args, **kwargs):
     get_redirect_route(
         r'%s/' % feconf.SUGGESTION_URL_PREFIX,
         suggestion.SuggestionHandler),
+    get_redirect_route(
+        r'%s/translation/<suggestion_id>' % feconf.SUGGESTION_URL_PREFIX,
+        suggestion.SuggestionHandler),
```

Comment:
Wh a different URL for the same handler?

Reply:
The prefix is same. But this change uses a `PUT` method which is in the same handler.

Reply:
This will allow `'%s/translation/<suggestion_id>'` to work for post method, right? Please use one URL for one handler.

Reply:
Initially I made this with a different handler. As a result of an offline discussion with Sagang, we decided to have this in the same handler using `PUT` method as it is an update functionality of the same flow of `SuggestionHandler`. @sagangwee would provide more context on this.

Reply:
It's fine to have it in the same handler, my question is why are we creating a new URL for it if we are going to use the same handler?